src/main/blackbox/blackbox.c

```diff
@@ -812,11 +812,7 @@ static bool sendFieldDefinition(const char * const *headerNames, unsigned int he
             return false; //Someone probably called us again after we had already completed transmission
         }
 
-        charsWritten = blackboxPrint("H Field ");
-        charsWritten += blackboxPrint(headerNames[xmitState.headerIndex]);
-        blackboxWrite(':');
-        charsWritten++;
-
+        charsWritten = blackboxPrintf("H Field %s:", headerNames[xmitState.headerIndex]);
```

Comment:
here is possible optimization for printf: https://github.com/cleanflight/cleanflight/commit/12a7ac9d5315a3c0bf2ca31e9b9d32361e01ad17

Format string and %s parameter are passed directly to `write(void* state, const char* data, int len)` function, rest of code is slightly optimized, only `%c` and `%%` are worse.